jobs/cjd-sample/quickstart/main.go

```diff
@@ -0,0 +1,44 @@
+// Copyright 2016 Google Inc. All rights reserved.
+// Use of this source code is governed by the Apache 2.0
+// license that can be found in the LICENSE file.
+
+// [START quick_start]
+package main
+
+import (
+	"fmt"
+	"log"
+
+	"golang.org/x/net/context"
+	"golang.org/x/oauth2/google"
+	jobs "google.golang.org/api/jobs/v2"
+)
+
+func main() {
```

Comment:
comment on the file path. `cjd-sample` is unnecessary. drop that directory.

maybe the top-level directory should be "jobdiscovery"?

Reply:
Later our product would be renamed from "job discovery" to "talent solution". But after all, our short name under google-api-services is always 'jobs'.